internal/validations/validations.go

```diff
@@ -60,6 +79,17 @@ func IsValidImageURI(imageURI string) bool {
 	return err == nil
 }
 
+// IsApprovedImageURI determines if the image URI is approved for use. Default is allow-all. If any container_uris are
+// set in the config, then provided input must be a direct match.
+func IsApprovedImageURI(imageURI string) bool {
+	if _, ok := imageURIs["*"]; ok {
+		return true
+	}
+
+	_, ok := imageURIs[imageURI]
```

Comment:
Wondering if this should be an exact match? Should there be globbing support (for example image registry, repos, tags, etc.)? If so, this probably impacts/takes care of what default is/how it's represented.

Reply:
Yeah, I took the short and sweet path, but I think at a minimum we need globbing to support any tag of a provided image. Might as well just properly support globbing as opposed to my all or nothing wildcard.